packages/trader/src/App/Components/Layout/Header/platform-switcher.jsx

```diff
@@ -0,0 +1,52 @@
+import classNames           from 'classnames';
+import PropTypes            from 'prop-types';
+import React                from 'react';
+import { CSSTransition }    from 'react-transition-group';
+import { localize }         from 'App/i18n';
+import Icon                 from 'Assets/icon.jsx';
+import { PlatformDropdown } from './platform-dropdown.jsx';
+import 'Sass/app/_common/components/platform-switcher.scss';
+
+class PlatformSwitcher extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = { is_open: false };
+    }
+
+    handleClick = () => this.setState({ is_open: !this.state.is_open });
```

Comment:
If it's a toggle it should be like this:
```suggestion
    toggle = () => this.setState(state => ({ is_open: !state.is_open }));
```

But maybe it would be better to be explicit in case we want to open/close specifically:
```suggestion
    setOpen = is_open => this.setState({ is_open });
```

Reply:
Done the changes and also made it more explicit since the dropdown only needs to close them, not open.